servicetalk-loadbalancer-experimental/src/main/java/io/servicetalk/loadbalancer/DefaultLoadBalancer.java

```diff
@@ -649,7 +651,7 @@ List<PrioritizedHostImpl<ResolvedAddress, C>> hosts() {
     }
 
     static final class PrioritizedHostImpl<ResolvedAddress, C extends LoadBalancedConnection>
-            implements Host<ResolvedAddress, C>, PrioritizedHost {
+            implements Host<ResolvedAddress, C>, PrioritizedHost, LoadBalancerObserver.Host {
```

Comment:
Consider adding `@Override` annotations in `io.servicetalk.loadbalancer.Host` that come from `LoadBalancerObserver.Host`

Reply:
Technically they're not related right now. Relating them will help make sure things like `isHealthy()` remain aligned, but conceptually I personally feel like keeping them distinct types makes sense as one is the state management machine and the other is some metadata for logging.
How strongly do you feel about it?

Reply:
Nvm, I read it as `io.servicetalk.loadbalancer.Host` extends `LoadBalancerObserver.Host` but now I see that only `PrioritizedHostImpl` implements both interfaces.